mirobo/strip.py

```diff
@@ -0,0 +1,69 @@
+from .device import Device
+from typing import Any, Dict
+import enum
+
+class Strip(Device):
+    """Main class representing the smart strip."""
+
+    def on(self):
+        """Power on."""
+        return self.send("set_power", ["on"])
+
+    def off(self):
+        """Power off."""
+        return self.send("set_power", ["off"])
+
+    def status(self):
+        """Retrieve properties."""
+        properties = ['power', 'temperature', 'current', 'mode']
+        values = self.send(
+            "get_prop",
+            properties
+        )
+        return StripStatus(dict(zip(properties, values)))
+
+    def set_power_mode(self, mode: PowerMode):
```

Comment:
The PowerMode is being used before it has been defined (see my comment in another PR :))